apps/vs-code-designer/src/app/utils/dotnet/dotnet.ts

```diff
@@ -240,32 +241,34 @@ export async function setDotNetCommand(context: IActionContext): Promise<void> {
     try {
       if (vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders.length > 0) {
         const workspaceFolder = await getWorkspaceFolder(context);
-        const projectPath = await tryGetFunctionProjectRoot(context, workspaceFolder);
+        const folderPath = isString(workspaceFolder) ? workspaceFolder : workspaceFolder.uri.fsPath;
```

Comment:
Just as a reminder on what we discussed. If you think this change will work better, I am good with it!

Reply:
Reverting